### PR TITLE
fix(deps): bump ejs to 3.1.8

### PR DESCRIPTION
**Pull request by bennetthardwick**
Currently when installing a CLI created with gluegun users will see a message saying "1 critical severity vulnerability" because of a vulnerability in ejs: https://github.com/advisories/GHSA-phwq-j96m-2c2q

While it's not likely this will cause an issue it might worry some people who install gluegun created CLIs.

This vulnerability is patched in ejs@3.1.7 so bumping the version will get rid of this message.

### CODE REVIEW

File: src/cli/templates/cli/package.json.ejs [hunk 1]
```diff
@@ -47,7 +47,7 @@
       "@typescript-eslint/eslint-plugin": "^4.17.0",
       "@typescript-eslint/parser": "^4.17.0",
       "ts-jest": "^26.5.3",
-      "ts-node": "^9.1.1",
+      "ts-node": "^10.9.1",
```
Comment (bennetthardwick) [inline src/cli/templates/cli/package.json.ejs:50]: @jamonholmgren this is a fix for the issue with CI happening in #763 (and all PRs I think). Let me know and I can split it out if you like.

**Comment by yulolimum** [comment]
@jamonholmgren we'll want to get this merged soon as it resolves issues on a freshly spun-up project. 

yulolimum-capture-2022-09-03--19-43-33

**Comment by StarlineBR** [comment]
If possible, update ejs to version 3.1.7 or higher as it is causing problems with windows powershell, by default powershell blocks execution of dependencies that have vulnerability and using script to bypass execution of vulnerable dependencies is not good for system security. **Thanks for the compression 💜 **

**Comment by StarlineBR** [comment]
If possible, update ejs to version 3.1.7 or higher as it is causing problems with windows powershell, by default powershell blocks execution of dependencies that have vulnerability and using script to bypass execution of vulnerable dependencies is not good for system security. **thanks for the attention and compression 💜**
unknown

**Comment by ThomasDRT** [comment]
Looking to see if we can get this merged and closed as well. We've got some workarounds in place but are looking forward to getting the vulnerability properly addressed. Thanks!

**Comment by danstepanov** [comment]
while `ejs` is no longer a dependency, the change to `ts-node` resolves this issue for me, thanks

**Pull request merged**